src/lib/Form/Data/ContentType/DraftContentTypesDeleteData.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\ContentType;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+
+class DraftContentTypesDeleteData
+{
+    /** @var ContentType[]|null */
+    protected $contentTypes;
+
+    /**
+     * @param ContentType[]|null $contentTypes
+     */
+    public function __construct(?array $contentTypes = [])
+    {
+        $this->contentTypes = $contentTypes;
+    }
+
+    public function getContentTypes(): ?array
+    {
+        return $this->contentTypes;
+    }
+
+    /**
+     * @param ContentType[]|null $contentTypes
+     */
+    public function setContentTypes(?array $contentTypes)
+    {
+        $this->contentTypes = $contentTypes;
+    }
+}
```

Comment:
Do you really need a second data structure identical to `\EzSystems\EzPlatformAdminUi\Form\Data\ContentType\ContentTypesDeleteData`? Why couldn't you re-use the existing one? :)
I mean, I see the comment about making dedicated classes, but this is too dedicated as it is the same.